Overview.bs

```diff
@@ -516,18 +516,36 @@ as follows:
 1.  If {{scrollSource}} is null, does not currently have a [=CSS layout box=], or
     if its layout box is not a [=scroll container=], return an unresolved time value.
 
-2.  Otherwise, let <var>current scroll offset</var> be the current scroll offset of {{scrollSource}}
+1.  Otherwise, let <var>current scroll offset</var> be the current scroll offset of {{scrollSource}}
     in the direction specified by {{orientation}}.
 
-3.  If <var>current scroll offset</var> is less than {{startScrollOffset}}, return an unresolved
+1.  If <var>current scroll offset</var> is less than {{startScrollOffset}}, return an unresolved
     time value if {{fill}} is <code>none</code> or <code>forwards</code>, 
     or 0 otherwise.
 
-4.  If <var>current scroll offset</var> is greater than or equal to {{endScrollOffset}}, 
-    return an unresolved time value if {{fill}} is <code>none</code> or
-    <code>backwards</code>, or the [=effective time range=] otherwise.
```

Comment:
It might be simpler to modify this condition here to return effective time range instead of unresolved, i.e.:
```
  4. If <var>current scroll offset</var> is greater than or equal to {{endScrollOffset}}, 
      return an unresolved time value if 
+     {{endScrollOffset}} is less than the maximum scroll offset
+     of {{scrollSource}} in {{orientation}} and
      {{fill}} is <code>none</code> or <code>backwards</code>
      or the [=effective time range=] otherwise.
```

With this, none of the additional text should be necessary.

Reply:
Fantastic idea, done!